content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md

```diff
@@ -0,0 +1,97 @@
+---
+title: HPE GreenLake for Private Cloud Enterprise (GLPCE) Deploying and scaling
```

Comment:
Please remove (GLCPE) from the title to be consistent with your first blog post of the series.

Reply:
done